tracer/src/Datadog.Trace/Agent/MessagePack/SpanMessagePackFormatter.cs

```diff
@@ -299,6 +283,26 @@ private int WriteTags(ref byte[] bytes, int offset, in SpanModel model, ITagProc
                 }
             }
 
+            // SCI tags will be sent only once per trace
+            if (model.IsLocalRoot || model.IsChunkOrphan)
```

Comment:
do we have the guarantee that the local root is always the first in the array of spans?
(the implementation expects the first span of the span array to contain the data and is not considering the trace tree stucture)

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/5082/commits/544bcd6a2ddb80dcdaa7833dd8cce45ea305bea3